app/lib/features/chat/widgets/custom_input.dart

```diff
@@ -884,55 +885,41 @@ class _TextInputWidgetConsumerState extends ConsumerState<_TextInputWidget> {
           }
         },
         onSubmitted: (_) => widget.onSendButtonPressed(),
-        style: Theme.of(context).textTheme.bodySmall,
+        style: Theme.of(context).textTheme.bodyMedium,
         decoration: InputDecoration(
+          fillColor: Theme.of(context).unselectedWidgetColor.withOpacity(0.5),
           contentPadding: const EdgeInsets.all(15),
           isCollapsed: true,
-          prefixIcon: widget.isEncrypted
-              ? Icon(
-                  Icons.shield,
-                  size: 18,
-                  color: Theme.of(context).colorScheme.primary.withOpacity(0.8),
-                )
-              : null,
-          suffixIcon: InkWell(
+          prefixIcon: InkWell(
             onTap: () => onSuffixTap(
               ref.read(chatInputProvider).emojiPickerVisible,
               context,
               ref,
             ),
             child: const Icon(Icons.emoji_emotions),
           ),
-          border: OutlineInputBorder(
-            borderRadius: BorderRadius.circular(30),
-            borderSide: BorderSide(
-              width: 0.5,
-              style: BorderStyle.solid,
-              color: Theme.of(context).colorScheme.surface,
-            ),
-          ),
+          hintText: widget.isEncrypted
+              ? L10n.of(context).newEncryptedMessage
+              : L10n.of(context).newMessage,
+          hintMaxLines: 1,
           focusedBorder: OutlineInputBorder(
-            borderRadius: BorderRadius.circular(30),
             borderSide: const BorderSide(
               width: 0.5,
-              style: BorderStyle.solid,
+              color: Colors.white38,
             ),
+            borderRadius: BorderRadius.circular(30),
           ),
           enabledBorder: OutlineInputBorder(
-            borderRadius: BorderRadius.circular(30),
-            borderSide: BorderSide(
+            borderSide: const BorderSide(
               width: 0.5,
-              style: BorderStyle.solid,
-              color: Theme.of(context).colorScheme.onSurface,
+              color: Colors.white30,
```

Comment:
what is our take on hardcoding colors now? I thought we'd not like to do that because it breaks e.g. dark-mode-switching ...